src/Emulator/Drawer/CreateHostView.tsx

```diff
@@ -150,6 +150,7 @@ export default function CreateHostView({ nameHint }: Props) {
       <FormControl>
         <FormLabel>IP Address</FormLabel>
         <SpaceSanitizedInput
+          fontFamily='monotype'
```

Comment:
Try `monospace` instead and see how that looks 🙂
```suggestion
          fontFamily='monospace'
```